target-platform/mylyn-docs.target

```diff
@@ -28,6 +28,12 @@
 					<version>31.1-jre</version>
 					<type>jar</type>
 				</dependency>
+				<dependency>
```

Comment:
The failureaccess jar is the only mandatory runtime dependency of guava.  Rather than specify includeDependencyDepth="direct" in the <location>, I figured it would make more sense to just specify this dependency explicitly.